### PR TITLE
DOC: Improved the docstring of pandas.Series.dt.to_pytimedelta

### PR DESCRIPTION
**Pull request by ijmbarr**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [X ] PR title is "DOC: update the <your-function-or-method> docstring"
- [X] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [X] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [X] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [X] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```################################################################################
################# Docstring (pandas.Series.dt.to_pytimedelta)  #################
################################################################################

Return an array of the Timedeltas in `datetime.timedelta` format with
the same length as the original Series.

Returns
-------
a : numpy.ndarray
    1D array containing data with `datetime.timedelta` type

Examples
--------
>>> s = pd.Series(pd.to_timedelta(np.arange(5), unit='d'))
>>> s
0   0 days
1   1 days
2   2 days
3   3 days
4   4 days
dtype: timedelta64[ns]
>>> s.dt.to_pytimedelta()
array([datetime.timedelta(0), datetime.timedelta(1),
       datetime.timedelta(2), datetime.timedelta(3),
       datetime.timedelta(4)], dtype=object)

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
        No summary found (a short summary in a single line should be present at the beginning of the docstring)
        See Also section not found

```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

No extended summary needed.

Checklist for other PRs (remove this part if you are doing a PR for the pandas documentation sprint):

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/indexes/accessors.py [hunk 1]
```diff
@@ -157,6 +157,30 @@ class TimedeltaProperties(Properties):
     """
 
     def to_pytimedelta(self):
+        """
+        Return an array of the Timedeltas in `datetime.timedelta` format with
```
Comment (cemsbr) [inline pandas/core/indexes/accessors.py]: Missing single-line summary in the beginning.

File: pandas/core/indexes/accessors.py [hunk 2]
```diff
@@ -157,6 +157,30 @@ class TimedeltaProperties(Properties):
...
+        the same length as the original Series.
+
+        Returns
+        -------
+        a : numpy.ndarray
+            1D array containing data with `datetime.timedelta` type
```
Comment (cemsbr) [inline pandas/core/indexes/accessors.py]: Full stop missing.

**Review by cemsbr — CHANGES_REQUESTED** [review]

File: pandas/core/indexes/accessors.py (hunk 2 above)
Comment (ijmbarr) [reply pandas/core/indexes/accessors.py]: Added.

File: pandas/core/indexes/accessors.py (hunk 1 above)
Comment (ijmbarr) [reply pandas/core/indexes/accessors.py]: Added.

File: pandas/core/indexes/accessors.py [hunk 3]
```diff
@@ -157,6 +157,32 @@ class TimedeltaProperties(Properties):
...
+        the same length as the original Series.
+
+        Returns
+        -------
+        a : numpy.ndarray
+            1D array containing data with `datetime.timedelta` type.
+
+        Examples
+        --------
+        >>> s = pd.Series(pd.to_timedelta(np.arange(5), unit='d'))
```
Comment (jreback) [inline pandas/core/indexes/accessors.py]: I *think* we need ``import datetime`` here

File: pandas/core/indexes/accessors.py [hunk 4]
```diff
@@ -157,6 +157,32 @@ class TimedeltaProperties(Properties):
...
+        >>> s
+        0   0 days
+        1   1 days
+        2   2 days
+        3   3 days
+        4   4 days
+        dtype: timedelta64[ns]
+        >>> s.dt.to_pytimedelta()
```
Comment (jreback) [inline pandas/core/indexes/accessors.py]: can you add a blank line before this case

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/core/indexes/accessors.py (hunk 4 above)
Comment (ijmbarr) [reply pandas/core/indexes/accessors.py]: Added.

File: pandas/core/indexes/accessors.py (hunk 3 above)
Comment (ijmbarr) [reply pandas/core/indexes/accessors.py]: I don't seem to need the import, but to clarify things, I've added more to the description.

**Review by cemsbr — APPROVED** [review]

**Review by jorisvandenbossche — APPROVED** [review]
Looks good!

**Comment by jorisvandenbossche** [comment]
Thanks @ijmbarr !

**Pull request merged**